google-cloud-clients/google-cloud-bigtable/synth.py

```diff
@@ -14,39 +14,112 @@
+
+
+def generate_admin_api(gapic):
+  library = gapic.java_library(
+      service='bigtable-admin',
+      version='v2',
+      config_path='/google/bigtable/admin/artman_bigtableadmin.yaml',
+      artman_output_name='')
+
+  # Excludes are relative to source `gapic-google-cloud-bigtable-v2/src`
+  excludes = [
+    "main/java/com/google/cloud/bigtable/admin/v2/package-info.java",
+  ]
+
+  # Paths are relative to the destination, which is the current working directory
+  internal_only = [
+    'src/main/java/com/google/cloud/bigtable/admin/v2/stub/GrpcBigtableInstanceAdminCallableFactory.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/stub/GrpcBigtableInstanceAdminStub.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/BaseBigtableInstanceAdminSettings.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/stub/GrpcBigtableTableAdminCallableFactory.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/stub/GrpcBigtableTableAdminStub.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/BaseBigtableTableAdminSettings.java',
+
+    'src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableInstanceAdminStub.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/stub/BigtableTableAdminStub.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/BaseBigtableInstanceAdminClient.java',
+    'src/main/java/com/google/cloud/bigtable/admin/v2/BaseBigtableTableAdminClient.java',
+  ]
+
+  s.copy(library / 'gapic-google-cloud-bigtable-admin-v2/src', 'src', excludes=excludes)
+  s.copy(library / 'grpc-google-cloud-bigtable-admin-v2/src', '../../google-api-grpc/grpc-google-cloud-bigtable-admin-v2/src')
+  s.copy(library / 'proto-google-cloud-bigtable-admin-v2/src', '../../google-api-grpc/proto-google-cloud-bigtable-admin-v2/src')
+
+  make_internal_only(internal_only)
+
+  java.format_code('../../google-api-grpc/grpc-google-cloud-bigtable-admin-v2/src')
+  java.format_code('../../google-api-grpc/proto-google-cloud-bigtable-admin-v2/src')
 
-admin_library = gapic.java_library(
-    service='bigtable-admin',
-    version='v2',
-    config_path='/google/bigtable/admin/artman_bigtableadmin.yaml',
-    artman_output_name='')
 
+def make_internal_only(sources):
+  """Annotates all toplevel classes with `@InternalOnly`"""
+  # Insert InternalOnly import in the non-static.
+  s.replace(sources,
+            before=r'^(import (?!static).*?\n)\n',
+            after=r'\1import com.google.api.core.InternalApi;\n\n',
+            flags=re.MULTILINE | re.DOTALL)
 
-s.copy(admin_library / 'gapic-google-cloud-bigtable-admin-v2/src', 'src')
-s.copy(admin_library / 'grpc-google-cloud-bigtable-admin-v2/src', '../../google-api-grpc/grpc-google-cloud-bigtable-admin-v2/src')
-s.copy(admin_library / 'proto-google-cloud-bigtable-admin-v2/src', '../../google-api-grpc/proto-google-cloud-bigtable-admin-v2/src')
+  # Replace javadoc and annotations before every public class with InternalApi
+  # javadoc and annotation
```

Comment:
Can you add the link to the feature request issue for the gapic generator?

Reply:
It's at the top of the file